caffe2/python/operator_test/pooling_test.py

```diff
@@ -145,6 +146,34 @@ def test_pooling_3d(self, stride, pad, kernel, size, input_channels,
            size=st.integers(7, 9),
            input_channels=st.integers(1, 3),
            batch_size=st.integers(1, 3),
+           **hu.gcs_gpu_only)
```

Comment:
Because MaxPoolWithIndex doesn't have a CPU implementation can you add the following before line 143
`@unittest.skipIf(not workspace.has_gpu_support, "No gpu support")`